Modelica/Electrical/Analog/Examples/Utilities/SwitchedCapacitor.mo

```diff
@@ -0,0 +1,75 @@
+within Modelica.Electrical.Analog.Examples.Utilities;
+model SwitchedCapacitor "Switched capacitor which can represent a positive or negative resistance"
+
+  parameter Modelica.SIunits.Time clock=1 "Clock";
+  parameter Modelica.SIunits.Resistance R=1 "Resistance";
+  Modelica.Blocks.Sources.BooleanPulse BooleanPulse(period=clock) annotation (Placement(transformation(extent={{-8,70},{12,90}})));
+  Modelica.Electrical.Analog.Basic.Capacitor Capacitor(C=clock/max(Modelica.Constants.eps,abs(R)))
```

Comment:
Does this pass the unit check?

Reply:
The exact unit-handling involving such cases is not yet fully standardized, and I will have to go through the details of the proposed MCP - but such cases *should* be handled.

Another solution is to just skip the guard - and just accept that bad parameter values lead to division by zero, the old models did not handle small resistances either (the difference was that it was specified using min-value - but here we want R to avoid a region around zero - which cannot be represented with min/max attribute).

Reply:
I like to keep the guard as I see this as an improvement so that should people end up using the new component don't end up with some error message just because of a 0 resistance. I hope that is acceptable for you.